presto-main/src/main/java/com/facebook/presto/memory/NodeMemoryConfig.java

```diff
@@ -24,11 +25,13 @@
 public class NodeMemoryConfig
 {
     public static final String QUERY_MAX_MEMORY_PER_NODE_CONFIG = "query.max-memory-per-node";
+    public static final long MAX_MEMORY = Runtime.getRuntime().maxMemory();
```

Comment:
Maybe `AVAILABLE_HEAP_MEMORY`? `MAX_MEMORY` is pretty vague, especially for a public final field.